TradeItIosTicketSDK2/TradeItErrorResult+ErrorCodeHelpers.swift

```diff
@@ -26,10 +26,6 @@ extension TradeItErrorResult {
     }
 
     func requiresAuthentication() -> Bool {
-        guard let integerCode = self.code?.integerValue
-            , let errorCode = TradeItErrorCode(rawValue: integerCode)
-            else { return false }
-
-        return errorCode == TradeItErrorCode.SESSION_ERROR || requiresRelink()
+        return !requiresRelink()
```

Comment:
We need to authenticate on all errors except the one which need to relink the account